pkgs/default.nix

```diff
@@ -5,8 +5,7 @@ let
   nixpkgs = builtins.fetchGit {
     name = "nixpkgs-20.03";
     url = "git@github.com:nixos/nixpkgs.git";
-    rev = "5272327b81ed355bbed5659b8d303cf2979b6953";
-    ref = "refs/tags/20.03";
+    rev = "3f690bfcd4adde6dd0733c2d9f8f4d61e09dfc60";
```

Comment:
Indicate in a comment, or in the name, that we’re not on the exact `nixpkgs-20.03` version ?

Reply:
Added `-snapshot` suffix.